base_layer/wallet/tests/output_manager_service_tests/service.rs

```diff
@@ -309,7 +309,11 @@ async fn fee_estimate() {
         &oms.key_manager_handle,
     )
     .await;
-    oms.output_manager_handle.add_output(uo, None).await.unwrap();
+    oms.output_manager_handle.add_output(uo.clone(), None).await.unwrap();
+    backend
+        .mark_output_as_unspent(uo.hash(&oms.key_manager_handle).await.unwrap())
+        .unwrap();
```

Comment:
Could these two calls not be combined into a helper function?

Reply:
Well its a 4 lines trimmed by fmt, 2 commands. Not that big a gain. 
As an bonus, you can see what you need to set to do this. 
I don't see any benefit to one.